strawberry_django/mutations/fields.py

```diff
@@ -177,10 +177,12 @@ def __init__(
         input_type: type | None = None,
         full_clean: bool = True,
         argument_name: str | None = None,
+        key_attr: str = "id",
```

Comment:
To fix the typing issues.

Also, pk because of the reasons I explained below

```suggestion
        key_attr: str | None = "pk",
```